File: python/7 kyu/312.py
# https://www.codewars.com/kata/58249d08b81f70a2fc0001a4/train/python

# Passed

def closest_multiple_10(i):
    remainder = i % 10
    if remainder < 5:
        result = i // 10 * 10
    else:
        result = (i // 10 + 1) * 10
    
    return result

output = closest_multiple_10(54)
print(output)

File: nodejs/7 kyu/312.js
// https://www.codewars.com/kata/5436f26c4e3d6c40e5000282/train/javascript

// Passed

function sumOfN(n) {
  let result = [];
  let sum = 0;
  let isNegative = n < 0;
  for (let i = 0; i < Math.abs(n) + 1; i++) {
    sum += i;
    result.push(sum * (isNegative ? -1 : 1));
  }

  return result;
}

const output = sumOfN(3);
console.log(output);